EIPS/eip-1.md

```diff
@@ -252,6 +252,24 @@ Permitted Networking Specifications URLs must anchor to a specific commit, and s
 ^https://github.com/ethereum/devp2p/blob/[0-9a-f]{40}/.*$
 ```
 
+### Bitcoin Improvement Proposal
+
+Links to Bitcoin Improvement Proposals may be included using normal markdown syntax, such as:
+
+```markdown
+[BIP 38](https://github.com/bitcoin/bips/blob/3db736243cd01389a4dfd98738204df1856dc5b9/bip-0038.mediawiki)
+```
+
+Which renders to:
+
+[BIP 38](https://github.com/bitcoin/bips/blob/3db736243cd01389a4dfd98738204df1856dc5b9/bip-0038.mediawiki)
+
+Permitted Bitcoin Improvement Proposal URLs must anchor to a specific commit, and so must match this regular expression:
+
+```regex
+^(https://github.com/bitcoin/bips/blob/[0-9a-f]{40}/bip-[0-9]+\.mediawiki)$
+```
+
```

Comment:
```suggestion

BIPs MUST be referenced in the non-hyphenated format `BIP X`, where X is the BIP number.

```

Reply:
None of the other permitted origins specify how to format the link text, so I'm leery about doing it for BIPs. I'd be perfectly happy to add an `eipw` lint to enforce this though.

Do you mean to prevent plain URL links (eg. `https://github.com/.../bip-0039.mediawiki`, `<https://github.com/.../bip-0039.mediawiki>`) as well, or just "if there is text, it must be `BIP XX`"?

Reply:
Don't RFCs specify formatting?

Reply:
> Don't RFCs specify formatting?

If you mean "do RFCs specify how to format links to RFCs", I suspect the answer is probably yes. We don't have an exception for RFCs yet though.